extensions/azurecore/src/account-provider/auths/azureAuth.ts

```diff
@@ -674,6 +678,11 @@ export interface Token extends AccountKey {
 	 */
 	token: string;
 
+	/**
+	 * Access token expiry timestamp
+	 */
+	azureAccountTokenExpiresOn?: number;
```

Comment:
I initially used `expiresOn` but switched to `azureAccountTokenExpiresOn` after seeing some other auth code path like dsts. I guess I can  change this one back to `expiresOn` since it's within `azureAuth`. Do you think same applies to AccountSecurityToken? I'm not sure if that one is Azure Auth specific.

Reply:
updated

Reply:
AccountSecurityToken is much more generic so is fine to keep I think, generally all tokens here are going to be for an account of some sort.